bundles/org.openhab.binding.goecharger/src/main/java/org/openhab/binding/goecharger/internal/handler/GoEChargerHandler.java

```diff
@@ -287,7 +301,9 @@ private void sendData(String key, String value) {
             if (logger.isDebugEnabled()) {
                 logger.debug("Response: {}", contentResponse.getContentAsString());
             }
-            updateStatus(ThingStatus.ONLINE, ThingStatusDetail.NONE);
+            GoEStatusResponseDTO result = gson.fromJson(contentResponse.getContentAsString(),
```

Comment:
You can now save `contentResponse.getContentAsString()` to a variable and also use it in the log statement above. Since the content string is now used anyways, it also makes sense to unconditionally log it without `if (logger.isDebugEnabled()) {` since it won't have an additional impact performance.

Reply:
fixed it for both get and post